liquibase-core/src/main/java/liquibase/diff/DiffResult.java

```diff
@@ -69,7 +68,7 @@ public Set<? extends DatabaseObject> getMissingObjects() {
     }
 
     public <T extends DatabaseObject> Set<T> getMissingObjects(Class<T> type) {
-        Set returnSet = new HashSet();
+        Set returnSet = new LinkedHashSet();
```

Comment:
I see that tests pass, but I don't understand how these changes make it :) i.e. even this part - where HashSet is changed to LinkedHashSet to preserve order looks strange as we anyway add values into it from missingObjects, which is also a HashSet, so even if we didn't use LinkedHashSet - shouldn't we just get the same order anyway?